tests/integration/base.py

```diff
@@ -124,22 +124,96 @@ def _create_session(cursor, session_id, user_id):
 
 def _create_global_topics(cursor):
     # type: (psycopg2.extensions.cursor) -> None
+
     cursor.execute(
-        "insert into"
-        " phpbb_topics(topic_id, forum_id, topic_title, topic_time,"
-        "  topic_first_poster_name, topic_first_post_id, topic_type)"
-        " values (0,0,'naslov_teme_0',10,'ime',0,3),"
-        " (1,0,'naslov teme 1',13,'drugi poster',1,3),"
-        " (2,0,'naslov teme 2',200,'post it',2,3),"
-        " (3,0,'naslov teme 3',256,'posted it',3,3)"
+        "insert into phpbb_topics ("
+        "topic_id"
+        ",forum_id"
+        ",topic_title"
+        ",topic_time"
+        ",topic_first_poster_name"
+        ",topic_first_post_id"
+        ",topic_type"
+        ") values ("
+        "0"
+        ",0"
+        ",'topic title 0'"
+        ",10"
+        ",'name'"
+        ",0,3"
+        "), ("
+        "1,"
+        "0,"
+        "'topic title 1',"
+        "13,"
+        "'second poster',"
+        "1,"
+        "3"
+        "), ("
+        "2"
+        ",0"
+        ",'topic title 2'"
+        ",200"
+        ",'post it'"
+        ",2,"
+        "3"
+        "), ("
+        "3"
+        ",0"
+        ",'topic title 3'"
+        ",256"
+        ",'posted it'"
+        ",3"
+        ",3"
+        "), ("
+        "4"
+        ",0"
+        ",'topic title 4'"
+        ",666"
+        ",'posted it again'"
+        ",3"
+        ",3"
+        ") , ("
+        "6"
+        ",0"
+        ",'topic title 5'"
+        ",777"
+        ",'posted as 7, expecting 6'"
+        ",3"
+        ",3"
+        ") , ("
+        "7"
+        ",0"
+        ",'different data type'"
+        ",999"
+        ",'inserted into table, expecting different type'"
+        ",3"
+        ",3"
+        ")"
         )
+
     cursor.execute(
-        " insert into "
-        " phpbb_posts(post_id,post_subject,post_text)"
-        " values (0,'prva tema', 'bla'),"
-        " (1,'druga tema','blabla'),"
-        " (2,'tretja tema','blablabla'),"
-        " (3,'tretja tema','bla krat 4')"
+        " insert into phpbb_posts ("
```

Comment:
Be careful!

You have `INNER JOIN` in the query. Which means other topics won't show up, because their posts do not exist!

Add posts before changing your tests to see if it will break your tests.

Reply:
Posts have been added in the next commit.